archive/r/r/longest-word.R

```diff
@@ -0,0 +1,13 @@
+longest_word_length <- function(input_string) {
+  # Checking input
+  args <- commandArgs(trailingOnly = TRUE)
+  
+  if (length(args) == 0 || args == "") {
+    cat("Usage: please provide a string\n")
+  } else {
+    # Split the string by whitespace and find the maximum word length in one line
+    cat(max(nchar(unlist(strsplit(args[1], "[ \t\n\r]+")))), "\n")
+  }
+}
+
+longest_word_length()
```

Comment:
This needs to accept a command-line argument. Please see the [project documentation](https://sampleprograms.io/projects/longest-word/)